diff --git a/public/apple-touch-icon.png b/assets/images/apple-touch-icon.png
similarity index 100%
rename from public/apple-touch-icon.png
rename to assets/images/apple-touch-icon.png
diff --git a/public/favicon-16x16.png b/assets/images/favicon-16x16.png
similarity index 100%
rename from public/favicon-16x16.png
rename to assets/images/favicon-16x16.png
diff --git a/public/favicon-32x32.png b/assets/images/favicon-32x32.png
similarity index 100%
rename from public/favicon-32x32.png
rename to assets/images/favicon-32x32.png
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 617c6c7..9c940c3 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -5,9 +5,9 @@
         <title>{% block title %}Mixed Vinyl{% endblock %}</title>
 
         <meta name="viewport" content="width=device-width, initial-scale=1">
-        <link rel="apple-touch-icon" sizes="180x180" href="{{ asset('apple-touch-icon.png') }}">
-        <link rel="icon" type="image/png" sizes="32x32" href="{{ asset('favicon-32x32.png') }}">
-        <link rel="icon" type="image/png" sizes="16x16" href="{{ asset('favicon-16x16.png') }}">
+        <link rel="apple-touch-icon" sizes="180x180" href="{{ asset('images/apple-touch-icon.png') }}">
+        <link rel="icon" type="image/png" sizes="32x32" href="{{ asset('images/favicon-32x32.png') }}">
+        <link rel="icon" type="image/png" sizes="16x16" href="{{ asset('images/favicon-16x16.png') }}">
 
         {% block stylesheets %}
             <link rel="stylesheet" href="{{ asset('styles/app.css') }}">
