diff --git a/templates/base.html.twig b/templates/base.html.twig
index 8626478..05ab59a 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -12,7 +12,7 @@
         {% endblock %}
 
         {% block javascripts %}
-            <script src="{{ asset('app.js') }}"></script>
+            <script type="module">import '{{ asset('app.js') }}';</script>
             <script src="https://kit.fontawesome.com/5a377fab5b.js" crossorigin="anonymous"></script>
         {% endblock %}
     </head>
