diff --git a/templates/base.html.twig b/templates/base.html.twig
index 798cf95..f14584b 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -16,7 +16,7 @@
 
         {% block javascripts %}
             {{ importmap() }}
-            <script src="https://kit.fontawesome.com/5a377fab5b.js" crossorigin="anonymous"></script>
+            <script defer src="https://kit.fontawesome.com/5a377fab5b.js" crossorigin="anonymous"></script>
         {% endblock %}
     </head>
     <body class="bg-gray-800 text-white">
