diff --git a/templates/admin/dashboard.html.twig b/templates/admin/dashboard.html.twig
index d3acf22..6a32d68 100644
--- a/templates/admin/dashboard.html.twig
+++ b/templates/admin/dashboard.html.twig
@@ -1,7 +1,13 @@
 {% extends 'base.html.twig' %}
 
+{% block stylesheets %}
+    {{ parent() }}
+
+    <link rel="stylesheet" href="{{ asset('styles/admin.css') }}">
+{% endblock %}
+
 {% block body %}
-    <div>
+    <div class="admin-wrapper">
         <h1 class="text-xl font-bold">Admin Dashboard</h1>
 
         <div class="my-5 bg-gradient-to-r from-cyan-500 to-blue-500 rounded h-64"></div>
