diff --git a/templates/vinyl/homepage.html.twig b/templates/vinyl/homepage.html.twig
index 2fb272a..5a84760 100644
--- a/templates/vinyl/homepage.html.twig
+++ b/templates/vinyl/homepage.html.twig
@@ -6,7 +6,6 @@
 <div class="px-4">
     <h1 class="text-3xl font-semibold text-center mb-5 mr-3">{{ title }} <i class="fa fa-edit pl-3 text-xs"></i></h1>
     <div {{ stimulus_controller('hello') }}></div>
-    {{ asset('@symfony/stimulus-bundle/loader.js') }}
     <div class="flex justify-center mt-5 bg-gray-600 p-5 rounded-xl">
         <div class="">
             {{ include('mix/_recordSvg.html.twig') }}
